### PR TITLE
MMSP-7075 (Mark) Add helpers to get the size and size on disk of a directory

### PR DESCRIPTION
**Pull request by dgolub**
@markbenvenuto 

#### QA Performed
* Verified that `GetDirectorySize` matches the size in the directory's properties dialog box
* Verified that `GetDirectorySizeOnDisk` matches the size on disk in the directory's properties dialog box for a compressed directory
* Verified that `GetDirectorySizeOnDisk` matches the size on disk in the directory's properties dialog box for an uncompressed directory

### CODE REVIEW

File: findfile.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+			}
+			totalSize += subdirSize
+		} else if (info.FileAttributes & FileAttributeCompressed) != 0 {
+			fileName := filepath.Join(dirName, info.FileName)
+			compressedSize, err := GetCompressedSize(fileName)
+			if err != nil {
+				return 0, err
+			}
+			totalSize += compressedSize
+		} else if info.FileSize%clusterSize != 0 {
```
Comment (markbenvenuto) [inline findfile.go]: space around modulo? I pretty sure gofmt does this

Comment (dgolub) [inline findfile.go]: No, I ran gofmt on this file, and it didn't change that.

Comment (markbenvenuto) [inline findfile.go]: You are right. Put
info.FileSize%clusterSize
in parenthesis, and gofmt will parse it better.

**Comment by markbenvenuto** [comment]
Done reviewing

**Comment by dgolub** [comment]
Merged ccf0f24.

**Pull request closed without merging**